pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/route/GetOptionsForActivityInstanceQuestionRoute.java

```diff
@@ -131,8 +131,10 @@ public RemoteAutoCompleteResponse handle(Request request, Response response) {
         List<PicklistOption> optionMatches = options.stream().filter(option -> option.getOptionLabel().toUpperCase()
                 .contains(autoCompleteQuery.toUpperCase())).collect(Collectors.toList());
 
-        //include options with allowDetails = true
-        optionMatches.addAll(options.stream().filter(option -> option.isDetailsAllowed()).collect(Collectors.toList()));
+        //include options with allowDetails=true if no matches
+        if (optionMatches.isEmpty()) {
```

Comment:
Include options with allowDetails=true if only NO matches/suggestions found for entered text.